component/client/settings/ClientSetting.tsx

```diff
@@ -104,21 +105,12 @@ class ClientSetting extends React.Component<any, IClientSettingState> {
 
   public reloadSettings() {
     this.setState({
-      dirty: false
-    });
-    const temp = this.state;
-    for (const key in temp) {
-      if (temp.hasOwnProperty(key) && key !== "dirty") {
-        storage.get(key).then((value) => {
-          const data: any = {};
-          data[key] = {
-            original: value,
-            updated: value,
-          };
-          this.setState(data);
-        });
+      dirty: false,
+      username: {
+        original: "",
```

Comment:
This is more or less house keeping. It is to keep our states clean when a reset is triggered.